unishox2.js

```diff
@@ -1335,6 +1335,8 @@ function unishox2_decompress_simple(input, len) {
   return unishox2_decompress(input, len, null, USX_HCODES_DFLT, USX_HCODE_LENS_DFLT, USX_FREQ_SEQ_DFLT, USX_TEMPLATES);
 }
 
-module.exports = {unishox2_compress, unishox2_compress_simple,
+if("module" in window){
```

Comment:
Hi, thanks for the research!! Since this is a public library and we want as much users as possible without compromising security, I would go with the second option you have pointed out.  I guess even if someone fakes module or window, this will only assign module.exports.